src/components/modeler/Modeler.vue

```diff
@@ -320,6 +320,16 @@ export default {
     highlightNode(node) {
       store.commit('highlightNode', node);
     },
+    blurFocusedScreenBuilderElement() {
+      const elementsToBlur = ['input', 'select'];
+      const shouldBlurActiveElement = elementsToBlur.some((element) => {
+        return document.activeElement && document.activeElement.toString().toLowerCase().includes(element);
```

Comment:
Apparently there is a [`tagName`](https://developer.mozilla.org/en-US/docs/Web/API/Element/tagName) property on elements:
```js
document.activeElement.toString() // "[object HTMLInputElement]"
document.activeElement.tagName // "INPUT"
```
That might simplify the code (not that it's not simple as-is, just a suggestion):
```js
const shouldBlurActiveElement = elementsToBlur.some((element) => {
  return document.activeElement && document.activeElement.tagName.toLowerCase() === element
});
```

If you capitalize the element names in the array, then it could also be:
```js
const shouldBlurActiveElement = elementsToBlur.includes(document.activeElement && document.activeElement.tagName);
```

Reply:
Aah brilliant @ `tagName`

Reply:
Updated :)